pkgs/development/python-modules/ufoprocessor/default.nix

```diff
@@ -26,6 +26,10 @@ buildPythonPackage rec {
     mutatormath
   ];
 
+  # no tests
```

Comment:
Tests can be run using `python Tests/tests.py` according to source.

Reply:
Can people just use pytest? 

Also this is you package. Maybe you should fix that in a PR not me?

Reply:
Sure can, wasn't really broken though, was it?

Reply:
It printed to the log that zero tests where run and I am just cleaning that up. Nothing else. If you don't feel like getting the tests working I just update the comment.

Reply:
Just give me a minute I'll have the PR done in a few minutes

Reply:
#109550